### ISSUE TITLE
List not up-to-date

### ISSUE THREAD

**Issue by ruurtjan** [ghostdogpr/zio-cheatsheet#5]
Hi Pierre,

I find myself googling this list every once in a while, but I noticed that it's not up-to-date anymore. For example, it doesn't contain `ZIO#filterOrFail`. Do you intend to maintain this repo?

**Comment by ghostdogpr** [comment]
Yeah sure. The list is not exhaustive but if there's something important we can add it. Wanna make a PR for it?

**Comment by ruurtjan** [comment]
I made a PR for the `filterOr` transformers. Could you take a look?
